packages/web-react/docs/stories/examples/ModalWithAccordion.stories.tsx

```diff
@@ -32,31 +33,41 @@ export const ModalWithAccordion = () => {
         {isOpen ? 'Close' : 'Open'} Modal
       </Button>
       <Modal id="ModalExample" isOpen={isOpen} onClose={handleClose}>
-        <ModalHeader>
-          <Button isSquare color="tertiary" onClick={toggleModal} aria-expanded={isOpen} aria-controls="#ModalExample">
-            <Icon name="close" />
-          </Button>
-        </ModalHeader>
-        <ModalBody>
-          <UncontrolledAccordion id="AccordionExample" defaultOpen="AccordionItemExample1">
-            <AccordionItem id="AccordionItemExample0">
-              <AccordionHeader slot={<Pill>3</Pill>}>Accordion Header #0</AccordionHeader>
-              <AccordionContent>{content}</AccordionContent>
-            </AccordionItem>
-            <AccordionItem id="AccordionItemExample1">
-              <AccordionHeader slot={<Pill>3</Pill>}>Accordion Header #1</AccordionHeader>
-              <AccordionContent>{content}</AccordionContent>
-            </AccordionItem>
-            <AccordionItem id="AccordionItemExample2">
-              <AccordionHeader>Accordion Header #2</AccordionHeader>
-              <AccordionContent>{content}</AccordionContent>
-            </AccordionItem>
-            <AccordionItem id="AccordionItemExample3">
-              <AccordionHeader slot={<Pill>3</Pill>}>Accordion Header #3</AccordionHeader>
-              <AccordionContent>{content}</AccordionContent>
-            </AccordionItem>
-          </UncontrolledAccordion>
-        </ModalBody>
+        <ModalDialog>
+          <>
```

Comment:
Is the React Fragment really needed here? 🤔 In component stories there is no usage of the Fragment.

Reply:
They must have been there originally because I just copied this. Anyway, it's fixed